packages/strapi-plugin-users-permissions/config/policies/permissions.js

```diff
@@ -97,14 +101,14 @@ module.exports = async (ctx, next) => {
       []
     );
 
-  if (!permission) {
+  if (!permissions.length) {
     return handleErrors(ctx, undefined, 'forbidden');
   }
 
   // Execute the policies.
-  if (permission.policy) {
+  if (permissions[0].policy) {
```

Comment:
Should probably look at a different solution for this, we want to avoid hard coding items in an array within the plugin.

Reply:
Yeah I understand, but since it checks the array is empty on line 109 it seems fine. I will make PR if you have better solution.